packages/xr/src/XRManagerExtended.ts

```diff
@@ -13,19 +13,21 @@ import { XRSessionState } from "./session/XRSessionState";
 export class XRManagerExtended extends XRManager {
   /** @internal */
   static _featureMap: Map<TFeatureConstructor<XRFeature>, XRFeatureType> = new Map();
+  static _listenersPool: IXRListener[][] = [];
 
   /** Input manager for XR. */
   override inputManager: XRInputManager;
   /** Session manager for XR. */
   override sessionManager: XRSessionManager;
   /** Camera manager for XR. */
   override cameraManager: XRCameraManager;
+  /** All initialized features at this moment. */
+  readonly features: XRFeature[] = [];
```

Comment:
Don't need to `declare module "@galacean/engine" {`?

Reply:
Done.